projects/plugins/beta/src/class-plugin.php

```diff
@@ -473,14 +473,14 @@ public function install_and_activate( $source, $id ) {
 		// Cleanup after previous version of the beta plugin.
 		if ( $this->plugin_slug() === 'jetpack' && file_exists( WP_PLUGIN_DIR . '/jetpack-pressable-beta' ) ) {
 			require_once ABSPATH . 'wp-admin/includes/file.php';
-			$creds = request_filesystem_credentials( site_url() . '/wp-admin/', '', false, false, array() );
+			$creds = request_filesystem_credentials( site_url() . '/wp-admin/', '', false, '', array() );
```

Comment:
As above, I'd probably use one of these:
```suggestion
			$creds = request_filesystem_credentials( site_url() . '/wp-admin/', '', false, '', null );
```

```suggestion
			$creds = request_filesystem_credentials( site_url() . '/wp-admin/' );
```